dashboard/state_aggregator.py

```diff
@@ -195,6 +197,35 @@ async def list_objects(self, *, option: ListApiOptions) -> dict:
         result.sort(key=lambda entry: entry["object_id"])
         return {d["object_id"]: d for d in islice(result, option.limit)}
 
+    async def list_runtime_envs(self, *, option: ListApiOptions) -> List[dict]:
+        """List all runtime env information from the cluster.
+
+        Returns:
+            A list of runtime env information in the cluster.
+            We don't have id -> data mapping like other API because runtime env
+            doesn't have unique ids.
+        """
+        replies = await asyncio.gather(
```

Comment:
What's the result when fail to get data from some agents? Can we guarantee reply the part data of alive agents?

Reply:
<img width="1590" alt="Screen Shot 2022-04-25 at 11 08 42 PM" src="https://user-images.githubusercontent.com/18510752/165233158-9fcc1d0a-49fe-4beb-8024-bbc02adff025.png">

this is the current proposal. It will be added in the next PR